pkg/render/intrusion_detection_test.go

```diff
@@ -38,15 +39,7 @@ var _ = Describe("Intrusion Detection rendering tests", func() {
 	It("should render all resources for a default configuration", func() {
 		esConfigMap := relasticsearch.NewClusterConfig("clusterTestName", 1, 1, 1)
 
-		component := render.IntrusionDetection(
-			nil,
-			nil,
-			&corev1.Secret{ObjectMeta: metav1.ObjectMeta{Name: render.TigeraKibanaCertSecret}},
-			&operatorv1.InstallationSpec{Registry: "testregistry.com/"},
-			esConfigMap, nil, notOpenshift, dns.DefaultClusterDomain, render.ElasticsearchLicenseTypeUnknown, notManagedCluster,
-			false,
-			&internalManagerTLSSecret,
-		)
+		component := render.IntrusionDetection(nil, nil, nil, &corev1.Secret{ObjectMeta: metav1.ObjectMeta{Name: render.TigeraKibanaCertSecret}}, &operatorv1.InstallationSpec{Registry: "testregistry.com/"}, esConfigMap, nil, notOpenshift, dns.DefaultClusterDomain, render.ElasticsearchLicenseTypeUnknown, notManagedCluster, false, &internalManagerTLSSecret, false, nil, nil, nil)
```

Comment:
[Cosmetic] Could we keep the same formatting ? It is harder to review this since we are exceeding the 80 chars limit per line.